lib/ocpp/v201/smart_charging.cpp

```diff
@@ -8,13 +8,17 @@
 #include "ocpp/v201/device_model.hpp"
 #include "ocpp/v201/enums.hpp"
 #include "ocpp/v201/evse.hpp"
+#include "ocpp/v201/messages/SetChargingProfile.hpp"
 #include "ocpp/v201/ocpp_types.hpp"
 #include "ocpp/v201/transaction.hpp"
 #include <algorithm>
+#include <exception>
 #include <iterator>
 #include <memory>
 #include <ocpp/v201/smart_charging.hpp>
 #include <optional>
+#include <stdexcept>
+#include <vector>
```

Comment:
Do you really need all of these includes here? `vector` is already used in the header file so it must be included somehow. `<memory>` is directly included in the header file so not needed here.

Reply:
Sorry, clangd adds imports very aggressively. Removed 👍🏾